lib/crudry_resolver.ex

```diff
@@ -133,28 +133,31 @@ defmodule Crudry.Resolver do
       AccountsResolver.delete_user(%{id: id}, info)
```

Comment:
So in this module the `custo_query` receives 2 arguments, and the resolver will transform it in a function that receives only one argument to be consumed by `Crudry.Query`.

We need to make this clear in the docs